protocol-tests/src/main/java/fixtures/dpgcustomization/implementation/DpgClientImpl.java

```diff
@@ -424,6 +424,32 @@ public PagedFlux<BinaryData> getPagesAsync(String mode, RequestOptions requestOp
                 nextLink -> getPagesNextSinglePageAsync(nextLink, requestOptionsForNextPage));
     }
 
+    /**
+     * Get pages that you will either return to users in pages of raw bodies, or pages of models following growup.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * {
+     *     received: String(raw/model) (Required)
+     * }
+     * }</pre>
+     *
+     * @param mode The mode with which you'll be handling your returned body. 'raw' for just dealing with the raw body,
+     *     and 'model' if you are going to convert the raw body to a customized body before returning to users.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return pages that you will either return to users in pages of raw bodies, or pages of models following growup
+     *     along with {@link PagedResponse}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    private PagedResponse<BinaryData> getPagesSinglePage(String mode, RequestOptions requestOptions) {
```

Comment:
I am not sure why these sync paging get suddenly generated (without sync-stack flag).

However these unused private method would get picked-up by spotbugs.

Spotbugs config in this lib lagged a few months from SDK repo. If you are sure SDK repo would be fine, you can ignore it. (and we will update the spotbugs config in CI)

```
[INFO] BugInstance size is 6
[INFO] Error size is 0
[INFO] Total bugs: 6
[ERROR] Low: Private method com.azure.analytics.purview.account.implementation.CollectionsImpl.listChildCollectionNamesNextSinglePage(String, RequestOptions) is never called [com.azure.analytics.purview.account.implementation.CollectionsImpl] At CollectionsImpl.java:[line 1104] UPM_UNCALLED_PRIVATE_METHOD
[ERROR] Low: Private method com.azure.analytics.purview.account.implementation.CollectionsImpl.listChildCollectionNamesSinglePage(String, RequestOptions) is never called [com.azure.analytics.purview.account.implementation.CollectionsImpl] At CollectionsImpl.java:[line 838] UPM_UNCALLED_PRIVATE_METHOD
[ERROR] Low: Private method com.azure.analytics.purview.account.implementation.CollectionsImpl.listCollectionsNextSinglePage(String, RequestOptions) is never called [com.azure.analytics.purview.account.implementation.CollectionsImpl] At CollectionsImpl.java:[line 1037] UPM_UNCALLED_PRIVATE_METHOD
[ERROR] Low: Private method com.azure.analytics.purview.account.implementation.CollectionsImpl.listCollectionsSinglePage(RequestOptions) is never called [com.azure.analytics.purview.account.implementation.CollectionsImpl] At CollectionsImpl.java:[line 658] UPM_UNCALLED_PRIVATE_METHOD
[ERROR] Low: Private method com.azure.analytics.purview.account.implementation.ResourceSetRulesImpl.listResourceSetRulesNextSinglePage(String, RequestOptions) is never called [com.azure.analytics.purview.account.implementation.ResourceSetRulesImpl] At ResourceSetRulesImpl.java:[line 1668] UPM_UNCALLED_PRIVATE_METHOD
[ERROR] Low: Private method com.azure.analytics.purview.account.implementation.ResourceSetRulesImpl.listResourceSetRulesSinglePage(RequestOptions) is never called [com.azure.analytics.purview.account.implementation.ResourceSetRulesImpl] At ResourceSetRulesImpl.java:[line 1290] UPM_UNCALLED_PRIVATE_METHOD
```

Reply:
Maybe change this line
https://github.com/Azure/autorest.java/blob/5919b519427a83134156f289d35617e64bd01334/javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java#L1302

to make `methodType == ClientMethodType.PagingSyncSinglePage` to `NOT_GENERATE`?